tools/docker/builder/openwrt/build.sh

```diff
@@ -111,7 +111,7 @@ main() {
             TARGET_PROFILE=DEVICE_NETGEAR_RAX40
             PRPLMESH_VARIANT="-dwpal"
             PRPL_FEED="https://git.prpl.dev/prplmesh/iwlwav.git^6265989f2c47f7b42f64d25f455443b87902582e"
-            INTEL_FEED="https://git.prpl.dev/prplmesh/feed-intel.git^e3eca4e93286eb4346f0196b2816a3be97287482"
+            INTEL_FEED="https://git.prpl.dev/prplmesh/feed-intel.git^6323abee35f2d8cd6263656e9226176cf1bcfca7"
```

Comment:
Seems to me that this should be done in the same commit that updates the prpl feed (nitpick).